src/mon/PGMap.cc

```diff
@@ -33,9 +33,9 @@ using std::vector;
 
 using ceph::bufferlist;
 using ceph::fixed_u_to_string;
-
-using TOPNSPC::common::cmd_getval;
-using TOPNSPC::common::cmd_getval_or;
+using ceph::common::cmd_getval;
+using ceph::common::cmd_getval_or;
+using ceph::common::cmd_putval;
```

Comment:
@rzarzynski `cmd_putval` worked because it had `CephContext` (actually `TOPNSPC::common::CephContext`) as an argument.
Why it does affect name resolution, I have no idea. @ronen-fr Is it a feature, or a bug?

Reply:
Still I don't understand why absence or presence `CephContext` would make any difference for `cmd_putval` lookup. Anyway, this existed even before the patch.